src/css/profile/mobile/common/subtab.less

```diff
@@ -92,9 +92,25 @@
 	}
 
 	&:not(.ui-sub-tab-static) {
+		overflow-x: scroll;
+		margin-left: 0;
+		margin-right: 0;
+		width: 100%;
+
+		&::-webkit-scrollbar {
+			display: none;
```

Comment:
Is there a case of scroll?
Is there a reason for none between display and visibility?

Reply:
On reference movie scrollbar does not display. The `visibility` will reserve place for scrollbar I think it is not necessary in this case. Do you think `visibility` will be better?

Reply:
> Is there a case of scroll?

You mean if I really need a `overflow-x: scroll`. It seemed right to me. Otherwise, you will have to click on the next items to get to the last items.

Reply:
I also agree with Tomasz. In that case scrollbar does not display.